ui/src/components/Orchestration/Orchestration.tsx

```diff
@@ -118,7 +116,7 @@ const Orchestration = () => {
         addNotification!,
         selectedChain,
         'makeCreateAndFundInvitation',
-        { chainName: selectedChain, denom: 'ubld' },
+        { chainName: selectedChain }, // adjust as needed
```

Comment:
oh... the contract required `denom` here now.

The codegen stuff I did handles that. Maybe it's worthwhile after all?

Reply:
Just put denom back here, didn't mean to remove it initially, I think I rebased wrong